docs/wiki/NOS3_Workflows.md

```diff
@@ -0,0 +1,16 @@
+# Workflows & VM Settings
```

Comment:
The 2 images that go with this page should be updated.  They show a very ancient version of the VM desktop.